test/integration_test.go

```diff
@@ -277,6 +277,26 @@ func (ts *IntegrationTestSuite) TestBasic() {
 	ts.assertMetricCountAtLeast("temporal_long_request", 3, "operation", "PollWorkflowTaskQueue")
 }
 
+func (ts *IntegrationTestSuite) TestBasic_InvalidUTF8() {
+	var expected []string
+	err := ts.executeWorkflow("test-basic", ts.workflows.Basic, &expected)
+	ts.NoError(err)
+	ts.EqualValues(expected, ts.activities.invoked())
+	// See https://grokbase.com/p/gg/golang-nuts/153jjj8dgg/go-nuts-fm-suffix-in-function-name-what-does-it-mean
```

Comment:
there is no `fm` suffix here?

Reply:
I'd thought I'd removed the test from this PR, let me take another look now that I've eaten lunch